docs/source/glossary.rst

```diff
@@ -85,6 +88,9 @@ This glossary provides definitions for terms and abbreviations you may encounter
    Node Allocation
       The process of reserving a set of nodes for a specific job, ensuring that the required resources are available for successful execution.
 
+   Non-interactive job
+      A non-interactive job is a task submitted to a compute cluster that runs without the need for real-time user input. These jobs are defined by scripts that specify all the parameters and commands required for execution and do not provide a direct interface for user interaction during execution.**
```

Comment:
I would just remove the ** and everything else looks okay. I might make mention that these are sbatch jobs just so people make the connection between a non-interactive job and the sbatch command.